applications/asset_tracker_v2/tests/gnss_module/CMakeLists.txt

```
@@ -17,6 +17,7 @@ cmock_handle(../../src/modules/modules_common.h )
 cmock_handle(${ZEPHYR_BASE}/../nrf/include/app_event_manager.h )
 cmock_handle(${ZEPHYR_BASE}/../nrf/subsys/app_event_manager/app_event_manager_priv.h )
 cmock_handle(${ZEPHYR_BASE}/../nrf/include/date_time.h )
+cmock_handle(${ZEPHYR_BASE}/../nrf/include/modem/lte_lc.h )
```

Use:
```suggestion
cmock_handle(${ZEPHYR_NRF_MODULE_DIR}/include/modem/lte_lc.h )
```

and please update the lines above and below to similar pattern when anyway touching this file.

Note: `${ZEPHYR_BASE}/../nrfxlib` --> `${ZEPHYR_NRFXLIB_MODULE_DIR}`